### PR TITLE
Updated UG and DG to reflect changes in Person commands

### PR DESCRIPTION
**Pull request by sanjevi13**
Edited UML diagrams in DG and edited commands in UG.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -97,9 +97,9 @@ How the `Logic` component works:
 1. The command can communicate with the `Model` when it is executed (e.g. to add a person).
```
Comment (Jarrett0203) [inline docs/DeveloperGuide.md:97]: This actually refers to the previous line but Github issues. Basically change `AddCommand` to `AddPersonCommand`. Also at line 110-113.
```suggestion
1. This results in a `Command` object (more precisely, an object of one of its subclasses e.g., `AddPersonCommand`) which is executed by the `LogicManager`.
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
...
-Format: `add i/STUDENT_ID n/NAME p/PHONE_NUMBER e/EMAIL [b/BOOK]…​`
+Format: `addPerson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add i/A0123456M n/Bob Tan p/91112222 e/btan@mail.com b/A_Book`
+* `addPerson i/A0123456M n/Bob Tan p/91112222 e/btan@mail.com b/A_Book`
```
Comment (Jarrett0203) [inline docs/UserGuide.md:83]:
```suggestion
* `addPerson n/Bob Tan p/91112222 e/btan@mail.com a/ 313, Jurong East Street 32 b/Harry Potter`
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
...
 
-### Deleting a person : `delete`
+### Deleting a person : `deletePerson`
 
 Delete a student given a student ID.
```
Comment (Jarrett0203) [inline docs/UserGuide.md:87]:
```suggestion
Deletes a person at the specified person list index.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
 
 ### Person Commands
 
-### Adding a user: `add`
+### Adding a user: `addPerson`
 
 Adds a user to the records
```
Comment (Jarrett0203) [inline docs/UserGuide.md:74]:
```suggestion
Adds a person to the ExpressLibrary.
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
...
-* `add i/A0123456M n/Bob Tan p/91112222 e/btan@mail.com b/A_Book`
+* `addPerson i/A0123456M n/Bob Tan p/91112222 e/btan@mail.com b/A_Book`
 
-### Deleting a person : `delete`
+### Deleting a person : `deletePerson`
 
 Delete a student given a student ID.
 
-Format: `delete STUDENT_ID`
+Format: `deletePerson STUDENT_ID`
```
Comment (Jarrett0203) [inline docs/UserGuide.md:89]:
```suggestion
Format: `deletePerson INDEX`
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
...
 
 * Deletes the person at the specified STUDENT_ID.
```
Comment (Jarrett0203) [inline docs/UserGuide.md:91]:
```suggestion
* Deletes the person at the specified person list index given by the librarian.
```

File: docs/UserGuide.md [hunk 7]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
...
 
-Format: `list`
+Format: `listPerson`
 
-### Editing a person : `edit`
+### Editing a person : `editPerson`
 
 
-[//]: # (Edits an existing person in the address book.)
+Edits an existing person in the address book.
```
Comment (Jarrett0203) [inline docs/UserGuide.md:107]:
```suggestion
Edits an existing person in the ExpressLibrary.
```

File: docs/UserGuide.md [hunk 8]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
...
-[//]: # (  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`)
+  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 
-[//]: # ()
-[//]: # (Examples:)
 
-[//]: # (* `find John` returns `john` and `John Doe`)
+Examples:
+
+* `findPerson John` returns `john` and `John Doe`
```
Comment (Jarrett0203) [inline docs/UserGuide.md:155]: Maybe we might want to change this example if findCommand is going to be updated.
```suggestion
* `findPerson John` returns `john` , `John Doe` and `johnston`.
```

File: docs/UserGuide.md [hunk 9]
```diff
@@ -170,11 +170,11 @@ Format: `listBook`
 
 ### Clearing all entries : `clear`
 
-_Details coming soon ..._
+[//]: # (_Details coming soon ..._)
 
-[//]: # (Clears all entries from the address book.)
+Clears all entries from the address book.
```
Comment (Jarrett0203) [inline docs/UserGuide.md:175]:
```suggestion
Clears all entries from the ExpressLibrary.
```

File: docs/UserGuide.md [hunk 10]
```diff
@@ -215,12 +215,12 @@ If your changes to the data file makes its format invalid, ExpressLibrary will d
```
Comment (Jarrett0203) [inline docs/UserGuide.md:215]: Could you help to remove line 191? I missed that, thanks.

File: docs/UserGuide.md [hunk 11]
```diff
@@ -68,93 +69,92 @@ ExpressLibrary is a **desktop app for managing library users and books, optimize
```
Comment (Jarrett0203) [inline docs/UserGuide.md:69]: This is for line 65, change `list` to `listPerson`.

**Review by Jarrett0203 — CHANGES_REQUESTED** [review]
Just some renaming and formatting to be fixed.

File: docs/UserGuide.md (hunk 10 above)
Comment (sanjevi13) [reply docs/UserGuide.md:215]: done

**Review by Jarrett0203 — APPROVED** [review]
LGTM

**Pull request merged**